### PR TITLE
 Improved error handling and code readability

### PR DESCRIPTION
**Pull request by RupeshHacker**
## Pull Request Template

**Description:**

- This PR aims to improve error handling and code readability in the main.go file.
- Changes include better error handling in the UploadHandler function and clearer return message formatting.
- The motivation behind these changes is to enhance code quality, maintainability, and user experience.

**Checklist:**

-   [ ] I have formatted my code using  `goimport`  and  `golangci-lint`.
-   [ ] All new code is covered by unit tests.
-  ✅ This PR does not decrease the overall code coverage.
-  ✅ I have reviewed the code comments and documentation for clarity.

### CODE REVIEW

**Comment by RupeshHacker** [comment]
Anyone please provide some feedback

File: examples/using-file-bind/main.go [hunk 1]
```diff
@@ -39,13 +39,12 @@ func UploadHandler(c *gofr.Context) (interface{}, error) {
...
-	if err != nil {
-		return nil, err
-	}
-
-	defer os.RemoveAll("tmp")
+	// Defer removing the temporary folder after the function exits
+	defer func() {
+		if err := os.RemoveAll("tmp"); err != nil {
+			fmt.Println("Error removing tmp folder:", err)
+		}
```
Comment (vikash) [inline examples/using-file-bind/main.go:46]: Instead of fmt.Println, we should be using `c.Logger`

Comment (vipul-rawat) [reply examples/using-file-bind/main.go:46]: @RupeshHacker using `c.Logger()` is a syntactical error, please use `Errorf() `or `Error()` methods provided by c.Logger

**Review by vipul-rawat — CHANGES_REQUESTED** [review]
Please fix syntax error on line 45

**Comment by vikash** [comment]
This is not required.

**Pull request closed without merging**